### PR TITLE
fix(bridge): Trigger change detection to show proper git settings on project change (#4781)

### LINKED ISSUE #4781: Settings screen is not updated when the project is changed
If you try to change the project via the dropdown in the header, the information is not updated (Git URL, user and token).

# Definition of Done
- [ ] The information in the settings screen is updated after the project is changed.

### PR DESCRIPTION
**Pull request by laneli**
Fixes #4781

### CODE REVIEW

**Review by Kirdock — CHANGES_REQUESTED** [review]
If you change the git upstream in one project and switch to another project that does not have a git upstream, the validation for the input fields is triggered.
image

**Comment by laneli** [comment]
I re-set now all form controls of the git upstream form - shouldn't occur anymore

File: bridge/client/app/_components/ktb-project-settings-git/ktb-project-settings-git.component.ts [hunk 1]
```diff
@@ -17,7 +17,11 @@ export class KtbProjectSettingsGitComponent implements OnInit {
   @Input()
   set gitData(gitData: GitData) {
     this.gitUrlControl.setValue(gitData.remoteURI || '');
+    this.gitUrlControl.markAsUntouched();
+    this.gitUrlControl.markAsPristine();
     this.gitUserControl.setValue(gitData.gitUser || '');
+    this.gitUserControl.markAsUntouched();
+    this.gitUrlControl.markAsPristine();
```
Comment (Kirdock) [inline bridge/client/app/_components/ktb-project-settings-git/ktb-project-settings-git.component.ts:24]: Wrong control. Change it to `gitUserControl`.

**Review by Kirdock — CHANGES_REQUESTED** [review]

File: bridge/client/app/_components/ktb-project-settings-git/ktb-project-settings-git.component.ts (hunk 1 above)
Comment (laneli) [reply bridge/client/app/_components/ktb-project-settings-git/ktb-project-settings-git.component.ts:24]: Yes :D

**Review by Kirdock — APPROVED** [review]

**Pull request merged**